### PR TITLE
created users_collabs join table

### PR DESCRIPTION
**Pull request by KlitiHamataj**
@AlexandreDidion @timdavhen @LaureneEpp

### CODE REVIEW

File: app/models/collab.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class Collab < ApplicationRecord
   validates :name, presence: true, uniqueness: true
-  has_many :users_collab, dependent: :destroy
+  has_many :users_collabs, dependent: :destroy
+  has_many :users, through: :users_collabs, dependent: :destroy
```
Comment (AlexandreDidion) [inline app/models/collab.rb]: not sure to understand what the second dependent destroy should do

File: Gemfile [hunk 2]
```diff
@@ -46,10 +46,13 @@ group :development do
   gem 'web-console', '>= 3.3.0'
   gem 'listen', '~> 3.2'
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
+  gem "better_errors"
```
Comment (AlexandreDidion) [inline Gemfile:49]: what are those?

File: app/models/collab.rb (hunk 1 above)
Comment (KlitiHamataj) [reply app/models/collab.rb]: I'm not very clear about dependant: :destroy  in general but if you delete a user it should delete the collab also

File: Gemfile (hunk 2 above)
Comment (KlitiHamataj) [reply Gemfile:49]: It changes the default rails errors to a better interface. It makes it easier to debug

File: app/models/collab.rb (hunk 1 above)
Comment (AlexandreDidion) [reply app/models/collab.rb]: Yes but then the dependent: :destroy should be on the side of the user model no? 
That line I have the impression it's saying : when you delete a collab, delete his users

Comment (KlitiHamataj) [reply app/models/collab.rb]: yes, you are right, it's on collab model

Comment (LaureneEpp) [reply app/models/collab.rb]: Yes, I would remove the line 4

Comment (KlitiHamataj) [reply app/models/collab.rb]: no the line 4 shouldn't be removed, just the dependent destroy

**Pull request merged**